openpype/hosts/maya/plugins/publish/collect_look.py

```diff
@@ -77,9 +77,14 @@ def node_uses_image_sequence(node):
     node_path = get_file_node_path(node).lower()
 
     # The following tokens imply a sequence
-    patterns = ["<udim>", "<tile>", "<uvtile>", "u<u>_v<v>", "<frame0"]
-
-    return (cmds.getAttr('%s.useFrameExtension' % node) or
+    patterns = ["<udim>", "<tile>", "<uvtile>",
+                "u<u>_v<v>", "<frame0", "<f4>"]
+    try:
+        extension = cmds.getAttr('%s.useFrameExtension' % node)
+    except ValueError:
+        extension = None
```

Comment:
This should be `extension = False` to still return a boolean.
Also, might be better as a variable named `use_frame_extension`?